### ISSUE TITLE
How do you pronounce pueue?

### ISSUE THREAD

**Issue by asheliahut** [Nukesor/pueue#194]
I am generally curious on the proper pronunciation.

I have gone back and forth on how to say this
- pyew (like queue but just p)
- pah way way (the fun way)

**Comment by Nukesor** [comment]
The first one :D

There's actually a (kind of dumb) history behind this name.
I actually started working on this in 2015, when I was still at university :D

Back then, I basically only knew Python, which is why the first version of [Pueue was written in Python](https://github.com/Nukesor/pueue-python-legacy). \
When i started looking for a name, I went for the obvious... It's a queue... for shell commands... written in Python... \
Ingenious, am I right :D?

Anyhow, I still think that this is kind of funny and I kept the name for historical reasons after the rewrite :)

(I also like to think about `pew pew`)
